dev/tidb-in-kubernetes/deploy/tidb-operator.md

```diff
@@ -89,29 +101,33 @@ kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/
 kubectl get crd tidbclusters.pingcap.com
 ```
 
-`TidbCluster` 自定义资源创建后，可以在 Kubernetes 集群上安装 TiDB Operator。
+创建 `TidbCluster` 自定义资源类型后，接下来在 Kubernetes 集群上安装 TiDB Operator。
 
-> **注意：**
->
-> `<chartVersion>` 在后续文档中代表 chart 版本，例如 `v1.0.0`。
+1. 获取你要安装的 `tidb-operator` chart 中的 `values.yaml` 文件：
 
-获取你要安装的 `tidb-operator` chart 中的 `values.yaml` 文件：
+    {{< copyable "shell-regular" >}}
 
-{{< copyable "shell-regular" >}}
+    ```shell
+    mkdir -p /home/tidb/tidb-operator && \
+    helm inspect values pingcap/tidb-operator --version=<chart-version> > /home/tidb/tidb-operator/values-tidb-operator.yaml
+    ```
 
-```shell
-mkdir -p /home/tidb/tidb-operator && \
-helm inspect values pingcap/tidb-operator --version=<chartVersion> > /home/tidb/tidb-operator/values-tidb-operator.yaml
-```
+    > **注意：**
+    >
+    > `<chart-version>` 在后续文档中代表 chart 版本，例如 `v1.0.0`，可以通过 `helm search -l tidb-operator` 查看当前支持的版本。
 
-配置 `/home/tidb/tidb-operator/values-tidb-operator.yaml` 文件中的 `scheduler.kubeSchedulerImage` 为你的 Kubernetes 集群中的镜像。
+2. 配置 TiDB Operator
 
-{{< copyable "shell-regular" >}}
+    TiDB Operator 里面会用到 `k8s.gcr.io/kube-scheduler` 镜像，如果下载不了该镜像，可以修改 `/home/tidb/tidb-operator/values-tidb-operator.yaml` 文件中的 `scheduler.kubeSchedulerImage` 为 `registry.cn-hangzhou.aliyuncs.com/google_containers`。
```

Comment:
@lilin90 For English document, this step is unnecessary. Only the first and third steps are needed.

Reply:
@tennix Got it. All other changes apply to the English docs except this step 2?

Reply:
Yes.